docs/src/transforms.md

```diff
@@ -54,6 +54,16 @@ This transform will take a query, serialise it into a CQL4 compatible format and
 - CassandraSinkSingle:
     # The IP address and port of the upstream cassandra node/service.
     remote_address: "127.0.0.1:9042"
+
+    # When this field is provided TLS is used when connecting to the remote address.
+    # Removing this field will disable TLS.
+    tls:
```

Comment:
I'd have this commented out so it can be copied and used immediately.

Reply:
Same for the other examples with TLS configs.

Reply:
Commenting it out does make it harder to read the tls example.
But you make a good point and I think commenting it out is the right move.